src/style.css

```diff
@@ -2,0 +1,85 @@
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
```

Comment:
Same issue here with the `font-family` property in the `font-face` rule. It should only include the name of the font you're importing. (CSS issue)